scaphandre/README.md

```diff
@@ -0,0 +1,68 @@
+# Agent Check: Scaphandre
+
+## Overview
+
+This check monitors [Scaphandre][1].
+
+## Setup
+
+### Installation
+
+To install the Scaphandre check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/python/)
+ on any machine.
```

Comment:
Please make this a reference link and specify which developer toolkit it is in the link text

Reply:
Done--the actual toolkit depends on the user's platform (Mac vs Windows), so I added a little note about that too.